Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -100,6 +100,18 @@ theorem IsVonNBounded.union {s₁ s₂ : Set E} (hs₁ : IsVonNBounded 𝕜 s₁
 
 end Zero
 
+section ContinuousAdd
+
+variable [SeminormedRing 𝕜] [AddZeroClass E] [TopologicalSpace E] [ContinuousAdd E]
+  [DistribSMul 𝕜 E] {s t : Set E}
+
+protected theorem IsVonNBounded.add (hs : IsVonNBounded 𝕜 s) (ht : IsVonNBounded 𝕜 t) :
```

Comment:
Can we have `IsVonNBounded.sub`, `IsVonNBounded.neg`, `IsVonNBounded.smul` (or at least `IsVonNBounded.nsmul`, `IsVonNBounded.zsmul` if you can't get the general statement)?